test/parallel/test-console-is-a-namespace.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+// https://heycam.github.io/webidl/#es-namespaces
+// https://console.spec.whatwg.org/#console-namespace
+// https://github.com/w3c/web-platform-tests/blob/master/console/console-is-a-namespace.any.js
+
+const common = require('../common');
+const assert = require('assert');
+const { test, assert_equals, assert_true, assert_false } = common.WPT;
+
```

Comment:
Sorry for the confusion. I was referring to the ["WPT Refs" block](https://github.com/nodejs/node/blob/master/test/parallel/test-whatwg-url-searchparams-set.js#L8-L12) in the file I linked. Specifically, a permalink to the source of the file as well as a URL pointing to the license under which the WPT test is used should be included for compliance.